adapter/internal/oasparser/model/mgw_swagger.go

```diff
@@ -1244,5 +1244,9 @@ func (swagger *MgwSwagger) PopulateFromAPIYaml(apiYaml APIYaml) error {
 				endpointConfig.APIEndpointSecurity.Sandbox.Type)
 		}
 	}
+
+	if len(data.APIThrottlingPolicy) != 0 {
+		swagger.xWso2ThrottlingTier = data.APIThrottlingPolicy
```

Comment:
Have added this here https://github.com/wso2/product-microgateway/pull/2667/files#diff-f5f136bd49f2727c3794136db948bbe65b2d521f83119e55ef767fc8939b50c1L1128 . Removed the non-empty check because `swagger.xWso2ThrottlingTier` is a string type.

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2667/commits/cb8890eee5222009a84259410311d7384bef1664